src/App.js

```diff
@@ -28,31 +28,100 @@ const generateSquares = () => {
 const App = () => {
 
   const [squares, setSquares] = useState(generateSquares());
-
+  const [isXTurn, setXTurn] = useState(true);
+  const [winner, setWinner] = useState(null);
+  const [currentPlayer, setCurrentPlayer] = useState(PLAYER_1);
+ 
```

Comment:
This is a more advanced topic but I wanted to name that it's usually considered bad practice to have two variables that are used to track essentially the same thing (`isXTurn` and `currentPlayer` in this case).
This is because there's a risk of one variable getting changed without the other one being changed, causing the two to be inconsistent.